relay/common/src/lib.rs

```diff
@@ -0,0 +1,275 @@
+// Copyright 2019-2021 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Common runtime code for Polkadot and Kusama.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+pub mod assigned_slots;
+pub mod auctions;
+pub mod claims;
+pub mod crowdloan;
+pub mod elections;
+pub mod impls;
+pub mod paras_registrar;
+pub mod paras_sudo_wrapper;
```

Comment:
We should move these two to `polkadot-runtime-parachains`.

In general there are also other pallets part of this crate. I would personally not move them here, but I'm also not 100% convinced on my thinking :D

Reply:
Can we do this as follow up? I would like to restrict this MR to moving files, so that reviewing stays easy.  
But in general yes, IMHO these should go somewhere else.

Reply:
This should be done in Polkadot before, as they should stay in Polkadot. Then you also don't need to do here anything.

Reply:
Okay I can take a stab at it, if no one else does it before me.

Reply:
After some further internal discussion, we don't need this :see_no_evil:

Reply:
Okay. Then that folder should be good to go. Only excluded stuff is now: metrics, rococo, test-runtime and westend.  
And on the Cumulus side i will move the complete `parachains` folder in following that logic.